library/Fission/IPFS/Peer.hs

```diff
@@ -3,14 +3,21 @@ module Fission.IPFS.Peer
   , rawList
   , connect
   , fission
+  , getExternalAddress
   ) where
 
 import           RIO hiding (all)
 import qualified RIO.ByteString.Lazy as Lazy
 import qualified RIO.Text            as Text
 import           RIO.Process (HasProcessContext)
 
+import           Net.IPv4
```

Comment:
Consider `import qualified Net.IPv4 as IPv4`. It's shorter, but importantly less noisy / more focused on the goal

```haskell
-- Before --

Net.IPv4.decode

-- After --

IPv4.decode
```